core/src/main/java/org/elasticsearch/indices/recovery/RecoveryTargetHandler.java

```diff
@@ -39,11 +39,12 @@
     void prepareForTranslogOperations(int totalTranslogOps, long maxUnsafeAutoIdTimestamp) throws IOException;
 
     /**
-     * The finalize request clears unreferenced translog files, refreshes the engine now that
-     * new segments are available, and enables garbage collection of
-     * tombstone files.
-     **/
-    void finalizeRecovery();
+     * The finalize request clears unreferenced translog files, refreshes the engine now that new segments are available, enables garbage
```

Comment:
I don't think this is correct anymore - clean the translog files? (I know it was there before, it just caught my eye)